docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2071,7 +2071,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/storageProviders/<source>/location`|string|Object store location |
 |`fhirServer/bulkdata/storageProviders/<source>/endpointInternal`|string|Object store end point url used to read/write from COS |
 |`fhirServer/bulkdata/storageProviders/<source>/endpointExternal`|string|Object store end point url used in the constructed download URLs|
-|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory |
+|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory. It is recommended not to mount this to the root file system or the share a direct volume mount. For instance, if the volume is /output/bulkdata, use /output/bulkdata/data to ensure a failed mount does not result in writing to the root file system.|
```

Comment:
"or the share a direct volume mount" is awkward

Reply:
Updated to hopefully less awkward, but still valid, wording.